.github/workflows/release.yml

```
@@ -0,0 +1,31 @@
+name: release
+on:
+  release:
+    types: [ created ]
+
+permissions:
+  contents: write
+
+jobs:
+  create-release:
+    name: Release Go Binary
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [ linux, darwin ]
+        goarch: [ "386", amd64, arm64 ]
+        exclude:
+          - goarch: "386"
+            goos: darwin
+    steps:
+      - uses: actions/checkout@v3
+      - uses: wangyoucao577/go-release-action@v1
```

Oh and could you please use the commit hash of this action instead of the major tag version? This [prevents the maintainer of that action from changing the code in their action to something malicious.](https://julienrenaux.fr/2019/12/20/github-actions-security-risk/). It doesn't guarantee complete safety because downstream sub-dependency actions could still change, but it's better than just a version tag.

```suggestion
      - uses: wangyoucao577/go-release-action@b989099
```

In the future i'd like to make sure all actions are either stored in this repo, or are officially verified github actions. Running 3rd party actions to build security-sensitive software is a risky proposition. This is good for the meantime since Mnemonikey is still in the early testing/beta stages.